### PR TITLE
docs: fix some typos and grammar issue

### PR DESCRIPTION
**Pull request by jyizheng**
Fix some typos and grammar issues after I passed through README and CONTRIBUTING doc.

### CODE REVIEW

**Comment by jyizheng** [comment]
> in my opinion ,some changes are not necessary or even wrong

Which part is wrong and which is unnecessary?

**Comment by yunfeiyangbuaa** [comment]
flies？

**Comment by jyizheng** [comment]
> flies？

Yeah, you are right.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -135,11 +135,11 @@ Commit content represents all content changes included in one commit. We had bet
 * avoid very large change in a commit;
 * complete and reviewable for each commit.
 
-No matter commit message, or commit content, we do take more emphasis on code review.
+No matter what the commit message, or commit content is, we do take more emphasis on code review.
 
 ### PR Description
 
-PR is the only way to make change to Dragonfly project files. To help reviewers better get your purpose, PR description could not be too detailed. We encourage contributors to follow the [PR template](./.github/PULL_REQUEST_TEMPLATE.md) to finish the pull request.
+PR is the only way to make change to Dragonfly project flies. To help reviewers better get your purpose, PR description could not be too detailed. We encourage contributors to follow the [PR template](./.github/PULL_REQUEST_TEMPLATE.md) to finish the pull request.
```
Comment (allencloud) [inline CONTRIBUTING.md]: Why to change files to flies. I think `files` is the correct word. @jyizheng

**Comment by allencloud** [comment]
LGTM, thanks for your work. Wish to see you in the following contributions. 🌹 ❤️  @jyizheng

**Pull request merged**